spec/services/preserved_object_handler_update_version_spec.rb

```diff
@@ -249,26 +230,6 @@
         expect(pc).to have_received(:save!)
       end
 
-      it 'does not call PreservedObject.touch when PreservedCopy only has timestamp updates' do
```

Comment:
again save as in check_existence spec:  this test shows that PreservedCopy timestamps were updated for the check, but PreservedObject remained unchanged.  So test should stay, but be modified